src/main/groovy/com/github/j2objccontrib/j2objcgradle/tasks/J2objcTestTask.groovy

```
@@ -138,7 +138,7 @@ class J2objcTestTask extends DefaultTask {
                 "    testMinExpectedTests ${testCount}\n" +
                 "}\n"
         if (getTestMinExpectedTests() == 0) {
-            logger.warning "Min Test check disabled due to: 'testMinExpectedTests 0'"
+            logger.warn "Min Test check disabled due to: 'testMinExpectedTests 0'"
```

Was this silently failing?